dist/tools/usb-serial/ttys.py

```diff
@@ -52,16 +53,6 @@ def filters_match(filters, tty):
     return True
 
 
-def shorten(string, length):
```

Comment:
This is an unused leftover that I forgot to delete. Maybe I should do so in a separate commit.

Reply:
Done